cmd/convox/install_test.go

```diff
@@ -279,3 +281,36 @@ func TestReadCredentialsFromFile(t *testing.T) {
 	creds, err = readCredentialsFromFile("./data/fixtures/credswrong2.csv")
 	assert.EqualError(t, err, "credentials file is of unknown length")
 }
+
+// TestUrls checks that each URL returns 200.
+func TestUrls(t *testing.T) {
+	urls := [...]string{
+		iamUserURL,
+		"https://convox.com/docs/about-resources/",
+		"https://convox.com/docs/api-keys/",
+		"https://convox.com/docs/docker-compose-file/",
+		"https://convox.com/docs/troubleshooting/",
+		"https://convox.com/guide/balancers/",
+		"https://convox.com/guide/databases/",
+		"https://convox.com/guide/environment/",
+		"https://convox.com/guide/images",
+		"https://convox.com/guide/links/",
+		"https://convox.com/guide/one-off-commands/",
+		"https://convox.com/guide/services/",
+		"https://docs.docker.com/engine/installation/",
+		"https://docs.docker.com/engine/reference/builder/",
+		"https://git-scm.com/docs/gitignore",
+		"https://github.com/convox/release",
+		"https://guides.github.com/introduction/flow/",
+		"http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-custom-resources.html",
+	}
+
+	for _, url := range urls {
+		resp, err := http.Get(url)
+		assert.Nil(t, err)
```

Comment:
Small nitpick. We should us [`assert.NoError`](https://godoc.org/github.com/stretchr/testify/assert#NoError) when checking errors. Has the same affect just with a nicer output message if the assertion fails.